projects/ccf-rui/src/app/modules/content/content.component.ts

```diff
@@ -14,6 +14,7 @@ export class ContentComponent {
   /** HTML class name */
   @HostBinding('class') readonly clsName = 'ccf-content';
 
+  tutorialMode = true;
```

Comment:
Tie this into the page state. Also, there should be a button on the stage to turn on tutorial mode (I don't think its on by default). Check with spec.